common/components/UI/DomainsComponents/DomainModal/DomainForm/DomainStreets/index.tsx

```diff
@@ -3,29 +3,29 @@ import { validateField } from '@common/assets/features/validators'
 import { Select, Form } from 'antd'
 import { useGetAllStreetsQuery } from '@common/api/streetApi/street.api'
 
-const DomainStreets: React.FC = () => {
+interface DomainStreetsProps {
+  disabled?: boolean
+}
+
+const DomainStreets: React.FC<DomainStreetsProps> = ({ disabled = false }) => {
   const { data: streets, isLoading } = useGetAllStreetsQuery({})
 
   return (
     <Form.Item
       name="streets"
       label="Закріплені адреси"
-      rules={validateField('required')}
+      rules={!disabled && validateField('required')}
```

Comment:
This approach can cause a runtime error. Get rid of this, please. Leave `validateField('required')` only